poolapi/apijson.go

```diff
@@ -13,31 +13,32 @@ type Response struct {
 // TODO: make JSON tags lower-case and add "_" between words
 
 type PurchaseInfo struct {
-	PoolAddress   string `json:"PoolAddress"`
-	PoolFees      string `json:"PoolFees"`
-	Script        string `json:"Script"`
-	TicketAddress string `json:"TicketAddress"`
+	PoolAddress   string  `json:"PoolAddress"`
+	PoolFees      float64 `json:"PoolFees"`
```

Comment:
Well that is designed so users can copy/paste the value for poolfees= in dcrwallet.conf/ticketbuyer.conf.  Not sure why it would need to be changed.